store/filestore/file_cache.go

```diff
@@ -0,0 +1,52 @@
+package filestore
+
+import (
+	"sync"
+
+	"github.com/smancke/guble/store"
+)
+
+type cache struct {
+	entries []*cacheEntry
+	sync.RWMutex
+}
+
+func newCache() *cache {
+	fc := &cache{
+		entries: make([]*cacheEntry, 0),
+	}
+	return fc
+}
+
+func (fc *cache) Len() int {
```

Comment:
naming could be more consistent: 
filename is file_cache, struct is cache, variable name is fc